Formula/smug.rb

```diff
@@ -0,0 +1,17 @@
+class Smug < Formula
+  desc "Automate your tmux workflow"
+  homepage "https://github.com/ivaaaan/smug"
+  url "https://github.com/ivaaaan/smug/archive/refs/tags/v0.2.4.tar.gz"
+  sha256 "6417a3abcc2c8d6a30fd4f365421a314581100ba97b32e57c244a001d4617c90"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    system "smug"
```

Comment:
Can you check some output here?

Reply:
Sure thing, though it really just shows the help page. I'll look at other examples that check stdout.

Reply:
The formula now checks that the version reported by `smug` is correct